network-api/networkapi/donate/tests/test_landing_page.py

```diff
@@ -0,0 +1,50 @@
+from http import HTTPStatus
+
+from wagtail.models import Page as WagtailPage
+
+from networkapi.donate import factory as donate_factories
+from networkapi.donate import models as pagemodels
+from networkapi.wagtailpages.tests import base as test_base
+
+
+class FactoriesTest(test_base.WagtailpagesTestCase):
+    def test_page_factory(self):
+        donate_factories.DonateLandingPageFactory()
+
+
+class DonateLandingPageTest(test_base.WagtailpagesTestCase):
```

Comment:
Some basic tests for the `DonateLandingPage` factory and model

Reply:
Thanks for the helpful note. Let's add it in as inline comment for `DonateLandingPageTest`!

Reply:
Thanks for the suggestion Mavis! 

Instead of adding one inline comment for the `DonateLandingPageTest` instance, I added comments for all of the tests in the file, describing what they do.

My reasoning being if we were to add a comment saying "Some basic tests for the `BaseDonateLandingPage` model", this would probably become quickly outdated as we add more functionality and advanced tests to the file.

Reply:
Nice. I like this approach!